processing/src/test/java/org/apache/druid/segment/AutoTypeColumnIndexerTest.java

```diff
@@ -560,7 +560,7 @@ public void testNestedColumnIndexerSchemaDiscoveryNested() throws IndexSizeExcee
         () -> cursorList.get(1).getColumnSelectorFactory().makeDimensionSelector(dimensionSpec)
     );
     Assert.assertEquals(StructuredData.wrap(2L), valueSelector.getObject());
-    Assert.assertFalse(valueSelector.isNull());
+    Assert.assertNotNull(valueSelector.getObject());
```

Comment:
i think this can probably just be removed, the object of this indexer will never be null because the value is always wrapped in `StructuredData`